examples/annotation_import/image.ipynb

```diff
@@ -430,23 +432,26 @@
     {
       "metadata": {},
       "source": [
-        "# Identifying what values in the numpy array correspond to the mask annotation\n",
-        "color = (255, 255, 255)\n",
-        "mask_data = lb_types.MaskData(url=\"https://storage.googleapis.com/labelbox-datasets/image_sample_data/raster_seg.png\")\n",
+        "### Raster Segmentation (Byte string array)\n",
+        "url = \"https://storage.googleapis.com/labelbox-datasets/image_sample_data/raster_seg.png\"\n",
+        "# url = \"https://storage.googleapis.com/labelbox-datasets/image_sample_data/color_mask.png\"\n",
```

Comment:
do we want to leave the comments here?

Reply:
Does it make sense?  or not really and I should remove it?

Reply:
I mean looks like you have 
```
url = "https://storage.googleapis.com/labelbox-datasets/image_sample_data/color_mask.png"
# url = "https://storage.googleapis.com/labelbox-datasets/image_sample_data/color_mask.png"
```

^ these two lines in the notebook and I don't see why we need the seconds line. could be just a leftover?

Reply:
AH - will remove that

Reply:
Done